client/src/Search/index.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from "react";
+import Input from 'material-ui/Input';
 
 class Search extends Component {

```

Comment:
I think this component could be stateless. The state has to be stored at a higher level for anyone to access it.

Reply:
I think having state here makes sense. You can just pass in a prop that takes the state and updates the results. We're not reloading the results as they type in the search bar. Keeping the state internally will make it faster